primefaces/src/main/resources/META-INF/resources/primefaces/dialog/dialog.js

```diff
@@ -97,8 +97,9 @@ PrimeFaces.widget.Dialog = PrimeFaces.widget.DynamicOverlayWidget.extend({
     init: function(cfg) {
         this._super(cfg);
 
-        this.content = this.jq.children('.ui-dialog-content');
-        this.titlebar = this.jq.children('.ui-dialog-titlebar');
+        this.container = this.jq.find('.ui-dialog-container');
```

Comment:
Need JSDoc for `container` and update the description of `content` up top for our TypeScript check to pass.